cli/create-user.php

```diff
@@ -10,7 +10,7 @@
 }
 
 $usernames = listUsers();
-if (preg_grep("/^$username$/i", $usernames)) {
+if (is_bool(preg_grep("/^$username$/i", $usernames))) {
```

Comment:
I would prefer:

```suggestion
if (preg_grep("/^$username$/i", $usernames) !== false) {
```

and likewise for other similar cases below